<ui version="4.0" >
 <class>KWFrameRunaroundProperties</class>
 <widget class="QWidget" name="KWFrameRunaroundProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>326</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Layout of Text in Other Frames</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="runThrough" >
        <property name="text" >
         <string>Text will run through this frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="runAround" >
        <property name="text" >
         <string>Text will run around this frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="noRunaround" >
        <property name="text" >
         <string>Text will not run around this frame</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Run Around Side</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="left" >
        <property name="text" >
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="right" >
        <property name="text" >
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="longest" >
        <property name="text" >
         <string>Longest side</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Distance Between Frame and Text:</string>
       </property>
       <property name="buddy" >
        <cstring>distance</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KoUnitDoubleSpinBox" name="distance" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoUnitDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>KoUnitDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
